test/liquidation/liquidation.test.js

```diff
@@ -19,6 +20,12 @@ describe("Integration -- liquidation happy path", () => {
     tbtcToken = await TestToken.deploy()
     await tbtcToken.deployed()
 
+    const SignerBondsProcessorStub = await ethers.getContractFactory(
+      "SignerBondsProcessorStub"
+    )
+    signerBondsProcessor = await SignerBondsProcessorStub.deploy()
+    await signerBondsProcessor.deployed()
```

Comment:
Why don't we just use `SignerBondsEscrow`? It is a simple implementation and we want to assert ETH balance change anyway. 🤷

Reply:
See https://github.com/keep-network/coverage-pools/pull/39/commits/56e45d3fdb69f813aacf8f66f630e6288358f3ec